docs/filtering.md

```diff
@@ -5,7 +5,7 @@ title: Filtering tests
 
 MUnit provides several options for selecting what tests to run.
 
-## Run only a single test case
+## Run only a single te st case
```

Comment:
This is weird, how come scalafmt works on .md files? :O

Reply:
Nvm, this must have been me clicking the space by mistake while running the format :sweat_smile: